### PR TITLE
chore: configure delete-e2e-data's Makefile to build docker image

### PR DESCRIPTION
**Pull request by masaaania**
- added Dockerfile and Makefile to build and push the image.

### CODE REVIEW

File: hack/delete-e2e-data-mysql/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Run Command
 
 ```
-bazelisk run //hack/delete-e2e-data-mysql:delete-e2e-data-mysql -- delete \
+go run ./cmd/delete-e2e-data-mysql delete \
```
Comment (cre8ivejp) [inline hack/delete-e2e-data-mysql/README.md:4]: Please fix the package name to `hack`.

File: hack/delete-e2e-data-mysql/README.md [hunk 2]
```diff
@@ -11,3 +11,32 @@ bazelisk run //hack/delete-e2e-data-mysql:delete-e2e-data-mysql -- delete \
   --no-profile \
   --no-gcp-trace-enabled
 ```
+
+Delete data created by the e2e test whose test_id is `example`.
+
+```
+go run ./cmd/delete-e2e-data-mysql delete \
```
Comment (cre8ivejp) [inline hack/delete-e2e-data-mysql/README.md:18]: Same here.

File: hack/delete-e2e-data-mysql/README.md (hunk 1 above)
Comment (masaaania) [reply hack/delete-e2e-data-mysql/README.md:4]: oh, thanks

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**